evap/development/fixtures/test_data.json

```diff
@@ -153074,6 +153422,131 @@
     "cc_users": []
   }
 },
+{
+  "model": "evaluation.userprofile",
+  "pk": 954,
+  "fields": {
+    "is_superuser": false,
+    "password": "!dOl57Qs91TuNY39aO66wTs5TBRnuBhd3cWbk257M",
+    "last_login": null,
+    "email": null,
```

Comment:
Looks like these are the users you added as voters/participants to evaluation 1673 above.

The problem with these, if generated by model-bakery, is that they will have unrealistically empty fields for email, name, etc. We should have enough users in the test data already.

Can you remove these instances here and instead just pick some of the existing users for participants/voters above for 1673? (Should be solvable in a few minutes by just editing this json file, let us know if you get stuck)